test/controllers/observation_views_controller_test.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require("test_helper")
+
+# ------------------------------------------------------------
+#  Observation views - test updating the `reviewed` column
+# ------------------------------------------------------------
+class ObservationViewsControllerTest < FunctionalTestCase
+  def test_update
+    login("mary")
+    obs = Observation.needs_identification(users(:mary))
+    obs_count = obs.count
+
+    # Have to create the o_v, none existing
+    obs.take(5).pluck(:id).each do |id|
+      ObservationView.create({ observation_id: id,
+                               user_id: users(:mary).id,
+                               reviewed: true })
+    end
+
```

Comment:
TODO: This test needs to hit the controller.

Needs to send a bunch of `put(:update, params)` instead of directly changing OV, here

Reply:
Improved controller and fixed test.